Automattic/wp-calypso
# Signup: Deprecate ecommerce-design-first and design-first flow

Related to #

## Proposed Changes

Referred to the track event, `calypso_signup_start`, we can find that both `ecommerce-design-first` and `design-first` are never used for a long time, so this PR is focusing on removing all deps related to these flows.
* Remove the flow definition
* Remove unused steps: `site-type` and `site-title`
* Rename `getSiteTypePropertyDefaults` to `getSitePropertyDefaults` to keep the default values for creating a new site
* Remove the default theme definition since it's defined on the wpcom

## Testing Instructions



* Go to `/start`
* Ensure you're able to create a new site as before

## Pre-merge Checklist
